gc/base/MemorySubSpaceSemiSpace.cpp

```diff
@@ -421,15 +428,23 @@ MM_MemorySubSpaceSemiSpace::tearDown(MM_EnvironmentBase *env)
  ****************************************
  */
 void
-MM_MemorySubSpaceSemiSpace::flip()
+MM_MemorySubSpaceSemiSpace::flip(MM_EnvironmentBase *env, uintptr_t step)
 {
-	MM_MemorySubSpace *swapTemp = _memorySubSpaceSurvivor;
-	_memorySubSpaceSurvivor = _memorySubSpaceAllocate;
-	_memorySubSpaceAllocate = swapTemp;
+	if (1 == step) {
```

Comment:
In future changesets can you add an Enum or something similar for these values so they have a meaningful name? Thanks